### PR TITLE
[onert] Resolve svace for UNREACHABLE

### PR DESCRIPTION
**Pull request by YiHyunJin**
Removed unreachable points in the flow.

Signed-off-by: hj0412-yi <hj0412.yi@samsung.com>

### CODE REVIEW

**Comment by YiHyunJin** [comment]
Related : https://github.com/Samsung/ONE/issues/7111

File: runtime/onert/backend/gpu_cl/open_cl/MemoryManagement.cc [hunk 1]
```diff
@@ -111,7 +111,6 @@ absl::Status AssignObjectsToTensors(const std::vector<TensorUsageRecord<uint3>>
     default:
       return absl::InternalError("MemoryStrategy is not supported with current tensor size type.");
   }
-  return absl::OkStatus();
```
Comment (hseok-oh) [inline runtime/onert/backend/gpu_cl/open_cl/MemoryManagement.cc:114]: Above functions have same issue.

Comment (YiHyunJin) [reply runtime/onert/backend/gpu_cl/open_cl/MemoryManagement.cc:114]: Sorry i don't understand...
Can you explain a little more to me?

Comment (YiHyunJin) [reply runtime/onert/backend/gpu_cl/open_cl/MemoryManagement.cc:114]: ah.... line 99 and line 84?... oh Thanks!

**Comment by YiHyunJin** [comment]
@nnfw-bot test android-runtime-builder

**Review by chunseoklee — APPROVED** [review]

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**